src/components/layout/CyVerseAppBar.js

```diff
@@ -760,6 +771,16 @@ function CyverseAppBar(props) {
                             {adminDrawerItems}
                         </>
                     )}
+                    <Divider />
```

Comment:
Not a huge deal, but logged out users end up seeing 2 dividers in a row: 
![image](https://user-images.githubusercontent.com/8909156/105228212-88abb400-5b1f-11eb-983d-9f94f9ba458a.png)